### ISSUE TITLE
Computer Vision

### ISSUE THREAD

**Issue by SchwarzKatzeZwei** [SchwarzKatzeZwei/Keras-Code-samples#1]
- [x] Image classification from scratch
- [x] Simple MNIST convnet
- [x] Image segmentation with a U-Net-like architecture
- [x] 3D image classification from CT scans
- [x] Semi-supervision and domain adaptation with AdaMatch
- [x] Convolutional autoencoder for image denoising
- [x] OCR model for reading Captchas
- [x] Consistency training with supervision
- [ ] Next-Frame Video Prediction with Convolutional LSTMs
- [ ] CutMix data augmentation for image classification
- [ ] Grad-CAM class activation visualization
- [ ] Gradient Centralization for Better Training Performance
- [ ] Image Captioning
- [ ] Image classification via fine-tuning with EfficientNet
- [ ] Image classification with Vision Transformer
- [ ] Model interpretability with Integrated Gradients
- [ ] Keypoint Detection with Transfer Learning
- [ ] Knowledge Distillation
- [ ] Learning to Resize in Computer Vision
- [ ] Metric learning for image similarity search
- [ ] MixUp augmentation for image classification
- [ ] Image classification with modern MLP models
- [ ] Image classification with Perceiver
- [ ] Point cloud classification with PointNet
- [ ] RandAugment for Image Classification for Improved Robustness
- [ ] Few-Shot learning with Reptile
- [ ] Object Detection with RetinaNet
- [ ] Semantic Image Clustering
- [ ] Semi-supervised image classification using contrastive pretraining with SimCLR
- [ ] Image similarity estimation using a Siamese Network with a contrastive loss
- [ ] Image similarity estimation using a Siamese Network with a triplet loss
- [ ] Self-supervised contrastive learning with SimSiam
- [ ] Image Super-Resolution using an Efficient Sub-Pixel CNN
- [ ] Supervised Contrastive Learning
- [ ] Video Classification with a CNN-RNN Architecture
- [ ] Video Classification with Transformers
- [ ] Visualizing what convnets learn
- [ ] Pneumonia Classification on TPU
